metrics/validate_drp.yaml

```diff
@@ -10,6 +10,7 @@ PA1:
     doc: LPM-17
     url: http://ls.st/lpm-17
     page: 21
+  tags: photometry
```

Comment:
I'm a little afraid that your `photometry` and `astrometry` tags on the metrics are trying to solve a similar problem to the `chromatic` and `achromatic` tags I made on the specs.  I'm not sure if things need to be changed if everything is working, but I'd like to try to come to a consensus on best practices for this sort of thing.

I suspect we can use the same mechanism in both SQuaSH and the report generation if we think about it a little bit.

Reply:
Right let's discuss this further.  I guess you still need the `chromatic`/`achromatic` specification tag, suppose for instance you have an specification for an astrometric metric that depends on filter. I guess differential chromatic refraction could be an example of such. Then you would need both tags. Yet the `astrometry`/`photometry`/`image quality` metric tags are being used "classify" or "group" metrics, in this case for display purpose only.